documentation/specs/Badge.md

```diff
@@ -0,0 +1,186 @@
+# `Badge` Component Specification
+
+## Overview
+
+A Badge is a visual text label for small bits of supporting information.
+
+### Use Cases
+
+- Categorizing information
+- Getting a user's attention
+- Describe metadata
+
+### Features
+
+- Color coordinated
+- Exhaustive combination set of text and icons
+- Single size
+
+### Risks and Challenges
+
+- Minimizing confusion around accessibility when considering Badge contents along with an optional accessibility label
+- Keeping the API clear with all of the different possible states that the Badge can be in
+- Extending our theme tokens with the variants
+
+### Prior Art
+
+- [Polaris `<Badge />`](https://polaris.shopify.com/components/feedback-indicators/badge)
+- [Paste `<Badge />`](https://paste.twilio.design/components/badge)
+- [Spectrum `<Badge />`](https://react-spectrum.adobe.com/react-spectrum/Badge.html)
+- [Primer `<Label />`](https://primer.style/react/Label)
+
+---
+
+## Design
+
+A `Badge` is a relatively straightforward component. No dependencies on external libraries as it requires only basic markup. Our primitives library, React Aria, in fact doesn't have any hooks to support such a component due to its simplicity.
+
+### API
+
+```ts
+type BadgeVariant =
+  | "primary"
+  | "secondary"
+  | "black"
+  | "inverse"
+  | "gray"
+  | "positive"
```

Comment:
**Question:** Are we using positive/negative elsewhere? I thought I had mostly seen use use success/danger

Reply:
you're right! updated it to reflect what we've been using already in the lib